video_xblock/backends/youtube.py

```diff
@@ -178,10 +169,10 @@ def format_transcript_timing(sec, period_type=None):
             period_type (str): Timing period type (whether `end` or `start`).
         """
         # Get rid of overlapping periods.
-        if period_type == 'end' and float(sec) >= 0.001:
-            float_sec = float(sec) - 0.001
-            sec = float_sec
-        mins, secs = divmod(sec, 60)  # pylint: disable=unused-variable
+        float_sec = float(sec)
+        sec = float_sec - 0.001 if period_type == 'end' and float_sec >= 0.001 else sec
```

Comment:
This ternary operator is rather hard to comprehend.
Also, still unclear what's the fuzz with 0.001?